spec/models/rental.spec.js

```diff
@@ -0,0 +1,170 @@
+      Rental.findHistory(200, function(error, movie) {
+        expect(error).toEqual(null)
+        done()
+      })
+    })
+  })
+
+  describe("#findCurrent", function () {
+    it("should return current rentals for a customer", function (done) {
+      Rental.findCurrent(29, function (error, movie) {
+        expect(movie[0].id).toEqual(19)
+        done()
+      })
+    })
+
+    it('should throw an error if no rentals found', function (done) {
+      Rental.findCurrent(300, function(error, movie) {
+        expect(error).toEqual(null)
+        done()
+      })
+    })
+
+    it('if customer has no rentals', function (done) {
+      Rental.findCurrent(200, function(error, movie) {
+        expect(error).toEqual(null)
+        done()
+      })
+    })
+  })
+
+  describe("#findCurrentMovies", function () {
+    it("should return current customers currently renting a movie", function (done) {
+      Rental.findCurrentMovies('Die Hard', function (error, customers) {
+        expect(customers.length).toEqual(2)
+        expect(customers[0].id).toEqual(19)
+        done()
+      })
+    })
+
+    it('should throw an error if no rentals found', function (done) {
+      Rental.findCurrentMovies("bad title", function(error, movie) {
+        expect(error).toEqual(null)
+        done()
+      })
+    })
+  })
+
+  describe("#findHistoryMovies", function () {
+    it("should return rental history for a movie", function (done) {
+      Rental.findHistoryMovies('Die Hard', function (error, rentals) {
+        expect(rentals.length).toEqual(2)
+        expect(rentals[0].id).toEqual(29)
+        done()
+      })
+    })
+
+    it('should throw an error if no rentals found', function (done) {
+      Rental.findHistoryMovies("bad title", function(error, movie) {
+        expect(error).toEqual(null)
+        done()
+      })
+    })
+  })
+
+  describe("#find_customers", function () {
+    it("should return customers who rented a given movie", function (done) {
+      Rental.find_customers('Die Hard', function (error, customers) {
+        expect(customers.length).toEqual(2)
+        expect(customers[0].id).toEqual(19)
+        done()
+      })
+    })
+
+    it('should throw an error if no customers found', function (done) {
+      Rental.findHistoryMovies("bad title", function(error, movie) {
+        expect(error).toEqual(null)
+        done()
+      })
+    })
+  })
+
+  describe("#checkout", function () {
+    it("should checkout correct movie and decrease movie inventory when checked out", function (done) {
+      var preStock;
+      Rental.find('Speed', function(error, movie) {
+        preStock = movie[0].inventory
+      })
+      Rental.checkout("Speed", 29, function(error, movie){})
+      Rental.find('Speed', function(error, movie) {
+        var postStock = movie[0].inventory
+        expect(movie[0].title).toEqual('Speed')
+        expect(postStock).toEqual(preStock - 1)
+      })
+      done()
```

Comment:
Good job with the tests!! 
For a cleaner code I would recommend you to be consistence when you choose to use variables. in line 153 you used var postStock = movie[0].inventory. So I would expect that next you'll have also variable for the movie[0].title.
Or that you just won't use the postStock, and use the raw data in the test.

But other then that - looks great!